test/run_pass/test_net_http_outgoing_buffer.js

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2015-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var net = require('net');
+var http = require('http');
+var fs = require('fs');
+var assert = require('assert');
+
+var binaryMessage = fs.readFileSync('./resources/random_binarydata');
```

Comment:
Personally I would like to have a buffer created there with a bit of binary data (just to not depend on the fs module).
Example: `new Buffer([0x1, 0x2, 0x3, ...])`;

Reply:
Great idea